modules/dns/variables.tf

```diff
@@ -45,7 +45,9 @@ variable "dnssec_config" {
       { algorithm = "rsasha256", key_length = 1024 }
     )
   })
-  default = null
+  default = {
```

Comment:
we should add `nullable = false` if we don't want null; I kind of like null as the default, it matches what we do in many other modules which logically means "I don't have it since I don't care about it" :)